### PR TITLE
Prep for deployment

### PR DESCRIPTION
**Pull request by cappadona**
@georev All yours. Let me know what I missed.

- [x] Push image to `092831676293.dkr.ecr.us-east-1.amazonaws.com/folio/order-import-poc:latest`
- [x] Copy `.env` to `s3://ecs-env-files/folio/order-import-poc/.env`

### CODE REVIEW

**Review by georev — COMMENTED** [review]
So I've tried to run the image that is in ECR on ECS and locally with the .env in S3 and I'm getting different results than when I run the image I built locally the other day with docker-compose with the current .env from S3. The ECR image says
```
Environment variables

baseOkapEndpoint: null
permELocation: null
permLocation: null
fiscalYearCode: null
materialType: null
```
on the screen. The one I built the other day does not.

Here's what is up on ECS.
http://cul-folio-contrib.library.cornell.edu/order-import-poc/import
You'll need to be on VPN.
I was noticing something wonky with the health checks that was causing the container to keep redeploying. I think I got that sorted, but if you get a 503 try again in like 30 seconds or a minute or so.

**Comment by cappadona** [comment]
Sorry about that @georev. I think I have it sorted. Image remains unchanged, but I did push a new version of the `.env` to S3.

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,4 @@
+USE_SYSTEM_ENV=true
```
Comment (cappadona) [inline .env.example:1]: @fereira Are you planning on retaining this switch (fallback to `application.properties`) moving forward?

Comment (fereira) [reply .env.example:1]: I don't have to, but if I end up migrating from jetty to spring-boot that's the more standard (default) way of reading in properties.  For local development I prefer not to have to deploy as a container, and just to start the app using maven.

**Comment by cappadona** [comment]
@fereira Robbie has redeployed if you want to take another look:
http://cul-folio-contrib.library.cornell.edu/order-import-poc/import

**Review by fereira — APPROVED** [review]
looks okay.

**Comment by cappadona** [comment]
Thanks @fereira. We have an [early draft of the deployment documentation](https://confluence.cornell.edu/pages/viewpage.action?pageId=395451703) available in Confluence.

**Pull request merged**